setup/ubuntu/bootstrap.sh

```diff
@@ -157,6 +157,10 @@ apt-get install -y libffi-dev libssl-dev
 # MySQL dependencies:
 apt-get install -y libmysqlclient-dev
 
+# Microsoft SQL Server dependencies (`sudo` required):
+sudo apt-get install -y freetds-dev
```

Comment:
I had a look at this but I can't really see where it would go. I guess you'll need to do it.